pkg/types/validation/installconfig_test.go

```diff
@@ -904,6 +904,7 @@ func TestValidateInstallConfig(t *testing.T) {
 				return c
 			}(),
 		},
+		// TODO(crawford): add a test to validate that homogeneous clusters are enforced once an additional architecture is added
```

Comment:
Is this TODO for this PR or for future work?

Reply:
Future work. We are waiting on s390x builds for the latest branch. It works on 4.2, so I'm going to fix it there first.